core/request.py

```diff
@@ -546,22 +547,22 @@ def findAndCall(self, path: str, *args, **kwargs) -> None:
                 caller = caller[currpath]
                 if (("exposed" in dir(caller) and caller.exposed) or ("internalExposed" in dir(
                     caller) and caller.internalExposed and self.internalRequest)) and hasattr(caller, '__call__'):
-                    args = self.pathlist[idx:] + [x for x in args]  # Prepend the rest of Path to args
+                    args = self.path_list[idx:] + [x for x in args]  # Prepend the rest of Path to args
                     break
             elif "index" in caller:
                 caller = caller["index"]
                 if (("exposed" in dir(caller) and caller.exposed) or ("internalExposed" in dir(
                     caller) and caller.internalExposed and self.internalRequest)) and hasattr(caller, '__call__'):
-                    args = self.pathlist[idx - 1:] + [x for x in args]
+                    args = self.path_list[idx - 1:] + [x for x in args]
                     break
                 else:
                     raise (errors.NotFound("The path %s could not be found" % "/".join(
                         [("".join([y for y in x if y.lower() in "0123456789abcdefghijklmnopqrstuvwxyz"])) for x in
-                         self.pathlist[: idx]])))
+                         self.path_list[: idx]])))
             else:
                 raise (errors.NotFound("The path %s could not be found" % "/".join(
                     [("".join([y for y in x if y.lower() in "0123456789abcdefghijklmnopqrstuvwxyz"])) for x in
-                     self.pathlist[: idx]])))
+                     self.path_list[: idx]])))
```

Comment:
Can you make this better? It's stupid to do the same stuff with a comprehension filtering out any characters and raising an `NotFound`-exception twice. Please use a tuple here as well.